config-example.js

```diff
@@ -0,0 +1,35 @@
+{
+	/*
+	 * Malboxes Example Configuration File
+	 *
+	 * Uncomment a specific section of the file to trigger a particular feature.
+     *
+     * Paths should be written using forward slashes even on Windows.
+     * For ex: C:/Tools
```

Comment:
Yeah, tab vs spaces issue... It's fixed in the other branch I'm about to push so I won't fix it here in order to avoid conflicts.